docker-compose.yml

```diff
@@ -1,10 +1,21 @@
-version: '3'
+version: '3.0'
 services:
-  wps:
-    build: .
-    image: pcic/sandpiper
+  sandpiper-dev:
+    image: pcic/sandpiper:latest
+    container_name: sandpiper-dev
+    environment:
+      - PYWPS_CFG=/wps.cfg
     ports:
-      - "8101:5003"
+      - "30101:5003"
+    volumes:
+      - ./dev-component/wps.cfg:/wps.cfg
+    restart: always
+
+networks:
+  default:
+    external:
+      name: pcicbr0
```

Comment:
Is this where the container is getting access to the thredds server? Is it still possible to `docker-compose up` locally and still have access to thredds server?

Reply:
Yes this is where access is given. This compose file is not meant to be run locally because of the port and network bridge. That being said if you run the container on your machine while you are connected to the vpn the container will have acess to thredds.